src/rest-api/login/index.ts

```diff
@@ -0,0 +1,9 @@
+import * as apiDouble from './loginAPI.double';
+import * as apiReal from './loginAPI.real';
+import { LoginFunction } from './loginAPI.contract';
+import { config } from '../config';
+
+export const login: LoginFunction  = (config.useRealAPI) ?
+                  apiReal.login
+                  :
+                  apiDouble.login;
```

Comment:
Mock, fake, stub, double...

Just read in an article that the api we are doing is more a double, it just does everything the real rest api is doing but with test data.

Right now looking for a good post a lot of stuff appears but not the one I read, sorry.